missing-values.qmd

```diff
@@ -1,26 +1,26 @@
-# Missing values {#sec-missing-values}
+# Valores faltantes {#sec-missing-values}
 
 ```{r}
 #| echo: false
 #| results: asis
 
 source("_common.R")
-mensagem_capitulo_sem_traducao()
+
 ```
 
-## Introduction
+## Introdução
 
-You've already learned the basics of missing values earlier in the book.
-You first saw them in @sec-data-visualization where they resulted in a warning when making a plot as well as in @sec-summarize where they interfered with computing summary statistics, and you learned about their infectious nature and how to check for their presence in @sec-na-comparison.
-Now we'll come back to them in more depth, so you can learn more of the details.
+Você já aprendeu o básico sobre valores faltantes (*missing values*) no início deste livro.
+Você os viu pela primeira vez no @sec-data-visualization, onde resultaram em um aviso ao criar um gráfico, bem como na @sec-summarize, onde interferiram no cálculo de estatísticas sumarizadas, e aprendeu sobre sua natureza de propagação e como verificar sua presença na @sec-na-comparison.
+Agora voltaremos a eles com mais profundidade, para que você possa aprender mais detalhes.
 
-We'll start by discussing some general tools for working with missing values recorded as `NA`s.
-We'll then explore the idea of implicitly missing values, values that are simply absent from your data, and show some tools you can use to make them explicit.
-We'll finish off with a related discussion of empty groups, caused by factor levels that don't appear in the data.
+Começaremos discutindo algumas ferramentas gerais para trabalhar com valores faltantes registrados como `NA`s.
```

Comment:
Não sei se já foi apresentado em algum capítulo anterior, mas talvez seja legal explicar que `NA` significa `Not Available`